.buildkite/pipeline.yml

```diff
@@ -37,7 +37,7 @@ steps:
     branches: "!master !develop !release-*"
     command:
       - "python3 -m pip install towncrier"
-      - "python3 -m towncrier.check"
+      - "python3 -m towncrier.check --compare-with=origin/develop"
```

Comment:
This change seems like it should be separate from this PR.

Reply:
Yep, we should factor that out.

Reply:
https://github.com/matrix-org/matrix-appservice-slack/pull/258